src/sbvr-api/sbvr-utils.coffee

```diff
@@ -35,7 +34,8 @@ _.assign(exports, errors)
 	SqlCompilationError
 	TranslationError
 	UnsupportedMethodError
-	ForbiddenError
+	BadRequestError
+	HttpError
```

Comment:
Should we export all the HTTPErrors now?

Reply:
We already do, there is a `_.assign(exports, errors)` right above :)

Reply:
Ahhh, sorry missed that :D thought this was the list of the ones we would export

Reply:
I think they are all exported by `_.assign(exports, errors)`. This deconstructs the `errors` exports into the individual used once here.